guides/source/routing.md

```diff
@@ -553,38 +553,33 @@ In particular, simple routing makes it very easy to map legacy URLs to new Rails
 
 ### Bound Parameters
 
-When you set up a regular route, you supply a series of symbols that Rails maps to parts of an incoming HTTP request. Two of these symbols are special: `:controller` maps to the name of a controller in your application, and `:action` maps to the name of an action within that controller. For example, consider this route:
+When you set up a regular route, you supply a series of symbols that Rails maps to parts of an incoming HTTP request. For example, consider these routes:
 
 ```ruby
-get ':controller(/:action(/:id))'
+get 'photos(/:id)', to: :index
+get 'photos/show(/:id)', to: :show
 ```
 
-If an incoming request of `/photos/show/1` is processed by this route (because it hasn't matched any previous route in the file), then the result will be to invoke the `show` action of the `PhotosController`, and to make the final parameter `"1"` available as `params[:id]`. This route will also route the incoming request of `/photos` to `PhotosController#index`, since `:action` and `:id` are optional parameters, denoted by parentheses.
+If an incoming request of `/photos/show/1` is processed by this route (because it hasn't matched any previous route in the file), then the result will be to invoke the `show` action of the `PhotosController`, and to make the final parameter `"1"` available as `params[:id]`. These routes will also route the incoming request of `/photos` to `PhotosController#index`, since `:id` is an optional parameter, denoted by parentheses.
 
```

Comment:
This example seems a bit awkward because `photos/show/1` does not happen in our usual app code.

Can we change it to something like 

``` ruby
get 'photos(/:id)', to: :display
```

And explain the `photos/` as well as `photos/1` examples?